doc/waarp-r66/source/changes.rst

```diff
@@ -50,12 +50,17 @@ Correctifs
   ou de debug qui peuvent être évités (en conservant les Warnings et les Erreurs)
   via l'ajout dans le fichier de configuration ``logback.xml`` les paramètres
   suivants en tête des options :
+- [`#51 <https://github.com/waarp/Waarp-All/pull/51>`__]
+  Diminution de l'empreinte mémoire pour le cas des clients simples et diminution
+  de la mémoire côté serveur pour les parties Web et REST.
+  (issue [`#52 <https://github.com/waarp/Waarp-All/issues/52>`__])
 
 .. code-block:: xml
 
   <statusListener
     class="org.waarp.common.logging.PrintOnlyWarningLogbackStatusListener" />
 
+- Mise à jour des dépendances
```

Comment:
I propose to add separate entries in the changelog for the changes of default values introduced in #51 (IMO, this is important enough to be highlighted).

```suggestion

.. code-block:: xml

  <statusListener
    class="org.waarp.common.logging.PrintOnlyWarningLogbackStatusListener" />

- [`#51 <https://github.com/waarp/Waarp-All/pull/51>`__]
  Diminution de l'empreinte mémoire pour le cas des clients simples et diminution
  de la mémoire côté serveur pour les parties Web et REST.
  (issue [`#52 <https://github.com/waarp/Waarp-All/issues/52>`__])
- [`#51 <https://github.com/waarp/Waarp-All/pull/51>`__] Si aucun argument
  ``-Xms`` n'est passé à la JVM lors du démarrage, une valeur par défaut de
  1024Mo est appliquée.
- [`#51 <https://github.com/waarp/Waarp-All/pull/51>`__] Les valeurs par défaut
  des limitations de bande passante ont changées : La limitation globale par
  défaut est maintenant de 100Gbps, et celle par connexion est de 1Gbps (ces
  valeurs peuvent être ajustées dans les fichiers de configuration).
- [`#51 <https://github.com/waarp/Waarp-All/pull/51>`__] La valeur par défaut
  de la RAM maximale utilisée par les services WEB et REST a été abaissée à 1Go
  (au lieu de 4Go) (cette valeur peut être ajustée dans les fichiers de
  configuration).
- Mise à jour des dépendances
```

Reply:
Done